docs/opentelemetry.api.rst

```diff
@@ -0,0 +1,17 @@
+opentelemetry.api package
+=========================
+
+Subpackages
+-----------
+
+.. toctree::
+
+    opentelemetry.api.trace
+
+Module contents
+---------------
+
+.. automodule:: opentelemetry.api
+    :members:
```

Comment:
Since these options seem to be the same everywhere, they should be added to the configuration https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_default_options